packages/components-native/src/Typography/Typography.style.ts

```diff
@@ -58,6 +43,31 @@ export const typographyTokens: { [index: string]: TextStyle } = {
   displayRegularBlack: {
     fontFamily: "jobberpro-blk",
   },
+};
+
+const fonts = Platform.select({
```

Comment:
Maybe explain why we doing this for future us

Reply:
I added a comment. Not sure if it's the clearest comment, I'm accepting suggestions 😄

Reply:
It sounds good to me!